pkg/gardenlet/controller/managedseed/valueshelper_test.go

```diff
@@ -145,9 +145,7 @@ var _ = Describe("ValuesHelper", func() {
 				APIVersion: configv1alpha1.SchemeGroupVersion.String(),
 				Kind:       "GardenletConfiguration",
 			},
-			FeatureGates: map[string]bool{
-				string(features.ReversedVPN): false,
-			},
+			FeatureGates: map[string]bool{},
```

Comment:
To preserve the exact test case that is described in https://github.com/gardener/gardener/pull/7167#discussion_r1053013003

```suggestion
			FeatureGates: map[string]bool{
				"FooFeature": false,
			},
```